src/Symfony/Bundle/FrameworkBundle/Resources/config/profiling.php

```diff
@@ -35,5 +36,14 @@
                 param('profiler_listener.only_main_requests'),
             ])
             ->tag('kernel.event_subscriber')
+
+        ->set('console_profiler_listener', ConsoleProfilerListener::class)
+            ->args([
+                service('profiler'),
+                service('.virtual_request_stack'),
+                service('debug.stopwatch'),
```

Comment:
can be missing, isn't it? we can remove the subscriber tag in the DI extension to unregister this behavior when its missing